packages/react-scripts/config/paths.js

```diff
@@ -86,17 +89,15 @@ module.exports = {
   ownNodeModules: resolveOwn('node_modules'), // This is empty on npm 3
 };
 
-const ownPackageJson = require('../package.json');
-const reactScriptsPath = resolveApp(`node_modules/${ownPackageJson.name}`);
-const reactScriptsLinked =
-  fs.existsSync(reactScriptsPath) &&
-  fs.lstatSync(reactScriptsPath).isSymbolicLink();
-
-// config before publish: we're in ./packages/react-scripts/config/
-if (
-  !reactScriptsLinked &&
-  __dirname.indexOf(path.join('packages', 'react-scripts', 'config')) !== -1
-) {
+// config before publish: appDir is cra monorepo root or packages/react-scripts
```

Comment:
Can you clarify "appDir is cra monorepo root" case? I don't understand which situation it's describing.

Reply:
Ha, I guess that was only necessary before #3755, https://github.com/facebookincubator/create-react-app/pull/3755/files#r161380787.
Auto-detecting when to use the template is (or was?) surprisingly complicated.  Might be easier to maintain to just have an explicit option like "react-scripts build --use-template" or "react-scripts build --appDir=template".